avocado_vt/loader.py

```diff
@@ -107,7 +108,10 @@ def _get_parser(self):
 
     def get_extra_listing(self):
         if self.args.vt_list_guests:
-            guest_listing(self.args)
+            args = copy.copy(self.args)
+            args.vt_config = None
+            args.vt_guest_os = None
```

Comment:
If I understood this correctly, it makes the extra listing (the guest list) show all guests, no matter what's set in the config file given with `--vt-config` and in `--vt-guest-os`.

This reinforces the idea that we need a custom command for Avocado-VT.  Avocado has the concept of tests, so Avocado-VT should plug there and use the same interface to populate its own tests, as it does.  Now, Avocado doesn't have the concept of guest types, so we have been using a hack.  But, we can live with this for now, since it's properly documented:

```
  --vt-list-guests      Also list the available guests (this option ignores
                        the --vt-config and --vt-guest-os)
```